### PR TITLE
Add phone redirection

### PR DESCRIPTION
**Pull request by pectom**

### CODE REVIEW

File: src/components/login/editProfile/PhoneAgreementView.js [hunk 1]
```diff
@@ -20,18 +21,27 @@ const styles = StyleSheet.create({
   },
 });
 
-function PhoneAgreementView({ label }) {
+function PhoneAgreementView({ label, setError }) {
   const loginContext = useContext(LoginContext);
   const { user } = loginContext;
   const [agreement, setAgreement] = useState(
-    user.is_phone_visible === undefined ? true : user.is_phone_visible
+    user.is_phone_enabled === undefined ? true : user.is_phone_enabled
```
Comment (AnetaPociecha) [inline src/components/login/editProfile/PhoneAgreementView.js]: I do not understand this line.

File: src/components/pairOverlay/ParticipantNameView.jsx [hunk 2]
```diff
@@ -21,7 +21,7 @@ function ParticipantNameView({ user }) {
   return (
     <View style={styles.container}>
       <Text numberOfLines={10} style={styles.text}>
-        {`${user.first_name} ${user.last_name}`}
+        {`${user.first_name}\n${user.last_name}`}
```
Comment (AnetaPociecha) [inline src/components/pairOverlay/ParticipantNameView.jsx]: What is the motivation behind this change? I like original solution better.

File: src/components/map/ClusterMarker.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        <View
+          style={{
+            position: 'absolute',
+            top,
+            left: 0,
+            right: 0,
+            bottom: 0,
+            justifyContent: 'center',
+            alignItems: 'center',
+          }}
```
Comment (AnetaPociecha) [inline src/components/map/ClusterMarker.js]: Please use style sheet.

Comment (pectom) [reply src/components/map/ClusterMarker.js]: Of course, this is how it should be done. I will fix that.

File: src/components/login/editProfile/PhoneAgreementView.js (hunk 1 above)
Comment (pectom) [reply src/components/login/editProfile/PhoneAgreementView.js]: The assumption was that it should work both with the current backend and after adding is_phone_visible field to the user. When I was creating it the user didn't have this field, so I assumed that by default this field is supposed to be true.  After updating the backend, the is_phone_visible field should be the same as in the database
